app/tasks/raster_tile_cache_assets/raster_tile_cache_assets.py

```diff
@@ -154,6 +154,7 @@ async def raster_tile_cache_asset(
             source_reprojection_parent_jobs,
             max_zoom_resampling=PIXETL_DEFAULT_RESAMPLING,
             max_zoom_calc=max_zoom_calc,
+            use_resampler=max_zoom_calc is None,
```

Comment:
Based on slack chat this morning, would it be better to do this instead?
```python
use_resampler=calc is None,
```

or 
```python
use_resampler=(calc is None and max_zoom_calc is None),
```